### DISCUSSION TITLE
NFC tag scanned in pyscript

### DISCUSSION THREAD

**Discussion by Danahi** [custom-components/pyscript#305] [Q&A] [answered]
Does anyone know how to catch that an NFC tag is scanned via pyscript?

**Comment by JayNewstrom** [comment, marked as answer]
@event_trigger("tag_scanned", "tag_id == 'UUID_HERE'")
def jay_car_nfc_tag_scanned(tag_id=None, device_id=None):

**Reply by Danahi** [reply]
thanks!
